pkg/sdn/plugin/proxy.go

```diff
@@ -41,25 +43,34 @@ func NewProxyPlugin(pluginName string, osClient *osclient.Client, kClient *kclie
 	}
 
 	return &ovsProxyPlugin{
-		registry: newRegistry(osClient, kClient),
+		kClient:  kClient,
+		osClient: osClient,
 		firewall: make(map[string][]proxyFirewallItem),
 	}, nil
 }
 
 func (proxy *ovsProxyPlugin) Start(baseHandler pconfig.EndpointsConfigHandler) error {
 	glog.Infof("Starting multitenant SDN proxy endpoint filter")
 
+	cn, err := proxy.osClient.ClusterNetwork().Get(osapi.ClusterNetworkDefault)
+	if err != nil {
+		return fmt.Errorf("could not get network info: %s", err)
+	}
+	proxy.networkInfo, err = parseNetworkInfo(cn.Network, cn.ServiceNetwork)
+	if err != nil {
+		return err
+	}
```

Comment:
As @dcbw suggested, we could add a helper method to fetch and parse networkInfo.